crates/scroll/hardforks/src/hardfork.rs

```diff
@@ -0,0 +1,272 @@
+                    Some(0)
+                }
+                Self::Bernoulli => Some(3747132),
+                Self::Curie => Some(4740239),
+                Self::Darwin => Some(6075509),
+                Self::DarwinV2 => Some(6375501),
+            },
+        )
+    }
+
+    /// Retrieves the activation timestamp for the specified hardfork on the Scroll Sepolia testnet.
+    pub fn scroll_sepolia_activation_timestamp<H: Hardfork>(fork: H) -> Option<u64> {
+        match_hardfork(
+            fork,
+            |fork| match fork {
+                EthereumHardfork::Homestead |
+                EthereumHardfork::Byzantium |
+                EthereumHardfork::Constantinople |
+                EthereumHardfork::Petersburg |
+                EthereumHardfork::Istanbul |
+                EthereumHardfork::Berlin |
+                EthereumHardfork::London |
+                EthereumHardfork::Shanghai => Some(0),
+                _ => None,
+            },
+            |fork| match fork {
+                Self::EIP150Block | Self::EIP155Block | Self::EIP158Block | Self::Archimedes => {
+                    Some(0)
+                }
+                Self::Bernoulli => Some(1713175866),
+                Self::Curie => Some(1718616171),
+                Self::Darwin => Some(1723622400),
+                Self::DarwinV2 => Some(1724832000),
+            },
+        )
+    }
+
+    /// Retrieves the activation timestamp for the specified hardfork on the Scroll mainnet.
+    pub fn scroll_mainnet_activation_timestamp<H: Hardfork>(fork: H) -> Option<u64> {
+        match_hardfork(
+            fork,
+            |fork| match fork {
+                EthereumHardfork::Homestead |
+                EthereumHardfork::Byzantium |
+                EthereumHardfork::Constantinople |
+                EthereumHardfork::Petersburg |
+                EthereumHardfork::Istanbul |
+                EthereumHardfork::Berlin |
+                EthereumHardfork::London |
+                EthereumHardfork::Shanghai => Some(0),
+                _ => None,
+            },
+            |fork| match fork {
+                Self::EIP150Block | Self::EIP155Block | Self::EIP158Block | Self::Archimedes => {
+                    Some(0)
+                }
+                Self::Bernoulli => Some(1714358352),
+                Self::Curie => Some(1719994277),
+                Self::Darwin => Some(1724227200),
+                Self::DarwinV2 => Some(1725264000),
+            },
+        )
+    }
+
+    /// Scroll mainnet list of hardforks.
+    pub fn scroll_mainnet() -> ChainHardforks {
+        ChainHardforks::new(vec![
+            (EthereumHardfork::Homestead.boxed(), ForkCondition::Block(0)),
+            (EthereumHardfork::Dao.boxed(), ForkCondition::Never),
+            (Self::EIP150Block.boxed(), ForkCondition::Block(0)),
+            (Self::EIP155Block.boxed(), ForkCondition::Block(0)),
+            (Self::EIP158Block.boxed(), ForkCondition::Block(0)),
```

Comment:
These can be replaced by Tangerine and SpuriousDragon

Reply:
fixed [5aea8e8](https://github.com/scroll-tech/reth/pull/56/commits/5aea8e8f9a41db16b531d92a1995fc75e70fa828)